### PR TITLE
Adapt to change in siesta 1.4(-SNAPSHOT)

### PR DESCRIPTION
**Pull request by adreghiciu**

### CODE REVIEW

**Comment by adreghiciu** [comment]
https://bamboo.zion.sonatype.com/browse/NXO-OSSF29-1

**Comment by adreghiciu** [comment]
CI :+1:

**Comment by nabcos** [comment]
+1

**Comment by kellyrob99** [comment]
+1

File: nexus-webapp/src/main/webapp/js/ext/Ajax.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    if ( contentType === 'application/vnd.siesta-error-v1+json') {
+      response.siestaError = Ext.decode(response.responseText);
+    } else if ( contentType === 'application/vnd.siesta-validation-errors-v1+json') {
+      response.siestaValidationError = Ext.decode(response.responseText);
+    }
+  });
+
+  // Set default HTTP headers
+  Ext.lib.Ajax.defaultPostHeader = 'application/json; charset=utf-8';
+});
```
Comment (jdillon) [inline nexus-webapp/src/main/webapp/js/ext/Ajax.js:30]: Should add lots of documentation around any customization/configuration of default Ext components

**Pull request merged**